PostProcessing/Runtime/Utils/RuntimeUtilities.cs

```diff
@@ -205,6 +205,7 @@ public static void BlitFullscreenTriangle(Texture source, RenderTexture destinat
         // Assumes that both textures have the exact same type and format
         public static void CopyTexture(CommandBuffer cmd, RenderTargetIdentifier source, RenderTargetIdentifier destination)
         {
+            // TODO: Not sure if this works with texture arrays...
```

Comment:
According to the doc it should.

Reply:
Yep, I should delete this.  A quick investigation into the underlying graphics APIs seems to confirm this.